WordPress/src/main/java/org/wordpress/android/models/ReaderTag.java

```diff
@@ -74,10 +75,17 @@ public String getCapitalizedTagName() {
         if (tagName == null) {
             return "";
         }
-        // HACK to allow iPhone, iPad, iEverything else
-        if (tagName.startsWith("iP")) {
+        // If already uppercase, assume correctly formatted
+        if (Character.isUpperCase(tagName.charAt(0))) {
```

Comment:
Although it's unlikely, it is possible that tagName will be "" in which case `charAt(0)` will cause a null pointer exception.